### DISCUSSION TITLE
Its possible to configure it to translate from any language into Spanish

### DISCUSSION THREAD

**Discussion by mlopezdev** [chidiwilliams/buzz#720] [Q&A] [answered]
When I use it, it only translates from any language into English, is it possible to translate or configure it to translate into Spanish by default instead of English?

**Comment by chidiwilliams** [comment]
Currently, Whisper only supports English-to-English or X-to-English translations, but if there’s a new update that supports translating to Spanish. I’ll be sure to add them.

**Comment by raivisdejus** [comment, marked as answer]
Latest versions of Buzz also support translations using AI. For usage see https://chidiwilliams.github.io/buzz/docs/usage/translations
